go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -150,6 +152,7 @@ var DefaultQsConfig = TabletConfig{
 	TransactionTimeout:      30,
 	TxShutDownGracePeriod:   0,
 	MaxResultSize:           10000,
+	WarnResultSize:          10000,
```

Comment:
I think we should default to 0 and not print the log when 0. Effectively disabling it by default. It would be a bit much to warn and error at 10k rows I think.